packages/client/modules/summary/components/NewMeetingSummary.tsx

```diff
@@ -19,8 +21,10 @@ interface Props {
 }
 
 const query = graphql`
-  query NewMeetingSummaryQuery($meetingId: ID!) {
+  query NewMeetingSummaryQuery($meetingId: ID!, $first: Int!, $after: DateTime) {
```

Comment:
+1 I know it's out of scope of this PR, but having generic `$first` is not a good name for a variable here, because it requires some search to find out it's about notifications. It would be nicer to have `$notificationsFirst` and `$notificationsAfter` here, but that would require changes in a lot of other places.